lib/board.rb

```diff
@@ -0,0 +1,36 @@
+class Board
+  attr_accessor :grid
+  attr_reader :size
+  def initialize
+    @grid = Array.new(3) { Array.new(3, '-') }
+    @size = 3 * 3
+  end
+
+  def display_board
+    puts " #{grid[0][0]} | #{grid[0][1]} | #{grid[0][2]} "
+    puts " #{grid[1][0]} | #{grid[1][1]} | #{grid[1][2]} "
+    puts " #{grid[2][0]} | #{grid[2][1]} | #{grid[2][2]} "
```

Comment:
You should make this method return a string instead of printing it. You can have new lines within the string. _Puts_ the output from main.rb wherever the method is called.

Reply:
## Display board changes

- We removed the puts statements from the display board method